waku-bindings/src/general/mod.rs

```diff
@@ -442,21 +442,21 @@ impl RegexRepresentation for Encoding {
 #[derive(Clone, Debug, Eq, PartialEq)]
 pub struct WakuContentTopic {
     pub application_name: Cow<'static, str>,
-    pub version: usize,
+    pub version: Cow<'static, str>,
     pub content_topic_name: Cow<'static, str>,
     pub encoding: Encoding,
 }
 
 impl WakuContentTopic {
     pub const fn new(
         application_name: &'static str,
-        version: usize,
+        version: &'static str,
         content_topic_name: &'static str,
```

Comment:
Maybe `Into<Cow<'static, str>>` instead of `&'static str` ?

Then below `application_name.into()` instead of `Cow::Borrowed(application_name)`